packages/account/src/Constants/onfido-phrases.tsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { LocaleConfig } from 'onfido-sdk-ui';
```

Comment:
`onfido-sdk-ui` was removed, Is it still in use anywhere?

Reply:
i have removed this LocaleConfig  usage.
@amina-deriv, i think this file is not used anywhere, shall we delete it?

Reply:
if its not in mster, yes please remove this.